src/pages/HealthContent.module.css

```diff
@@ -1,5 +1,10 @@
 .healthContent {
-  background-image: url("../assests/images/healthcare.png");
+  background-image: linear-gradient(
```

Comment:
This is a white-to-white gradient, right? I guess it's a hack to put a transparent white background on top of the image?
I'd say this is pretty hacky, but I'm not sure what the intention is and a different fix may be even more weird. So if this is the best option, that's all good.

Reply:
this is a more straightforward way of adding a transparency:

.overlay { background-color: #FFF; opacity: 0.6; } 

you just need to make sure it is sized to cover the bg